src/vivliostyle/table.js

```diff
@@ -1705,6 +1705,9 @@ goog.scope(function() {
      */
     vivliostyle.table.TableRowLayoutConstraint = function(nodeContext) {
         RepetitiveElementsOwnerLayoutConstraint.call(this, nodeContext);
+
+        /** @type {Array.<{constraints: Array.<adapt.layout.FragmentLayoutConstraint>, breakPosition:adapt.vtree.NodeContext}>} */
+        var cellFragmentConstraints = [];
```

Comment:
Fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/348/commits/ca474fc2982f896b89fbea7ae600e4628ae44d5d).